docs/pages/reference/cli/tctl.mdx

```diff
@@ -2051,6 +2051,95 @@ prod       Done  2025-01-15 18:00:12 outside_window
 backup     Done  2025-01-14 20:00:31 outside_window
 ```
 
+## tctl plugins install awsic
+
+Install the AWS Identity Center integration.
+
+```code
+$ tctl plugins install awsic <flags>
+```
+
+### Flags
+
+| Name | Default Value(s) | Allowed Value(s) | Description |
```

Comment:
Should we have a column that says which flag is required and which is optional?

Reply:
I noticed the current pattern to mention "optional" flag is explaining that in the Description column. 
One random example I pulled from `tctl bots add` command from reference - https://goteleport.com/docs/reference/cli/tctl/#flags-9

I updated that in https://github.com/gravitational/teleport/pull/52878/commits/1b8fd2b00c88b062ed8e3c1277d28264cef7694a